php/Dockerfile

```diff
@@ -53,9 +53,17 @@ RUN apt-get update && \
     ln -s /usr/lib/x86_64-linux-gnu/ImageMagick-6.8.9/bin-Q16/MagickWand-config /usr/bin/ && \
     rm -rf /var/lib/apt/lists/*
 
-# Install Apcu, Igbinary, Imagick, Redis and ssh2
-RUN pecl install apcu igbinary imagick redis ssh2-1.1.2 && \
-    docker-php-ext-enable apcu igbinary imagick redis ssh2
+# Install Apcu, Igbinary, Imagick, Redis
+RUN pecl install apcu igbinary imagick redis && \
+    docker-php-ext-enable apcu igbinary imagick redis
+
+# Clone, build and install SSH2 extension from source
+RUN \
+    cd /tmp && \
+    git clone https://git.php.net/repository/pecl/networking/ssh2.git && \
```

Comment:
You could clone in the directory of your choice by adding the path as last parameter. It will avoid you to use `cd` instructions.

Reply:
Thanks, it has been fixed